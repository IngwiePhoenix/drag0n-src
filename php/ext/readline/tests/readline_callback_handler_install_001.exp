%Atesting: bool(true)

Warning: readline_callback_handler_install(): foobar! is not callable in %s on line %d
bool(false)

Warning: readline_callback_handler_install() expects exactly 2 parameters, 1 given in %s on line %d
NULL