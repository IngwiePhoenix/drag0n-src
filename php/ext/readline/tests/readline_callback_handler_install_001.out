bool(true)

Warning: readline_callback_handler_install(): foobar! is not callable in /Users/Ingwie/Work/drag0n-php/ext/readline/tests/readline_callback_handler_install_001.php on line 8
bool(false)

Warning: readline_callback_handler_install() expects exactly 2 parameters, 1 given in /Users/Ingwie/Work/drag0n-php/ext/readline/tests/readline_callback_handler_install_001.php on line 9
NULL